ee/desktop/runner/server/server.go

```diff
@@ -0,0 +1,141 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"net/http"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/kit/ulid"
+	"github.com/kolide/launcher/pkg/agent/types"
+)
+
+type RootServer struct {
+	server                *http.Server
+	listener              net.Listener
+	logger                log.Logger
+	desktopProcAuthTokens map[string]string
+	mutex                 sync.Mutex
+	knapsack              types.Knapsack
```

Comment:
Optional, but you can declare what you need in an interface. And not the whole knapsack